src/api/links.js

```diff
@@ -0,0 +1,11 @@
+import { BASE_API_URL } from "./base";
+
+export const getLinks = async (folderId = null) => {
+  let url = `${BASE_API_URL}/api/users/1/links`;
+  if (folderId) url += `?folderId=${folderId}`;
```

Comment:
<img width="1388" alt="스크린샷 2024-02-19 오후 10 10 11" src="https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/assets/53929065/9179a6e3-05a7-4b8b-a3cf-c175a7be4673">

Reply:
💊 
`URLSearchParams` API를 이용하여 querystring을 만들어주면 어떨까요? 👀 
```suggestion
  const params = new URLSearchParmas();
  if (folderId) params.append('folderId', folderId);
  const url = `${BASE_API_URL}/api/users/1/links`;
```

Reply:
new URLSearchParmas(); 여기가 이해가 안됩니당 ㅠ 
if (folderId) params.append('folderId', folderId); 어팬드를 했는데 이해가안됩니당..

Reply:
해당 내용과 비슷하지만 별개로 query-string이 여러개일 경우 잘 다루는 방법을 오늘 멘토링 시간에 다뤄보도록 하겠습니다!